### DISCUSSION TITLE
Error while sending data to socket

### DISCUSSION THREAD

**Discussion by VpkPrasanna** [deepgram/community#668] [General help] [answered]
Error

NoneType can't be used in 'await' expression

line which creates this error is

await self.socket.send(bytes_data)

here self.socket = <deepgram.transcription.LiveTranscription object at 0x1062e9ab0>

request id - 15c82008-cec2-46a6-8907-9d2b14f1e6b7

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by lukeocodes** [comment]
Is this error while using an SDK?

**Reply by VpkPrasanna** [reply]
yes using sdk , version 2.2.0

**Reply by lukeocodes** [reply]
Which SDK?

**Reply by VpkPrasanna** [reply]
python sdk

**Reply by VpkPrasanna** [reply]
hi @lukeocodes  any update on the issue raised ?

**Reply by davidvonthenen** [reply, marked as answer]
Take a look to see what you are passing into the function and also make sure that you are async/await'ing any required function calls on your side.
